crosschain-resolver/contracts/L1Resolver.sol

```diff
@@ -0,0 +1,123 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.17;
+
+import {EVMFetcher} from '@ensdomains/evm-verifier/contracts/EVMFetcher.sol';
+import {EVMFetchTarget} from '@ensdomains/evm-verifier/contracts/EVMFetchTarget.sol';
+import {IEVMVerifier} from '@ensdomains/evm-verifier/contracts/IEVMVerifier.sol';
+
+contract L1Resolver is EVMFetchTarget {
+    using EVMFetcher for EVMFetcher.EVMFetchRequest;
+    IEVMVerifier immutable verifier;
+    address immutable target;
+    uint256 constant COIN_TYPE_ETH = 60;
+    uint256 constant RECORD_VERSIONS_SLOT = 1;
+    uint256 constant VERSINABLE_ADDRESSES_SLOT = 3;
+    uint256 constant VERSINABLE_HASHES_SLOT = 4;
+    uint256 constant VERSINABLE_TEXTS_SLOT = 11;
+    uint8   constant RECORD_VERSIONS_REF = 0;
```

Comment:
It doesn't really make sense for this to be a constant.

Reply:
You mean just RECORD_VERSIONS_REF or all other constants?

Reply:
Just that one.